docs/apis/text-recognition.md

```diff
@@ -118,11 +179,64 @@ Here we show how to visualize the [BoundingBox](text-recognition-api-ref.md#micr
 > [!NOTE]
 > For this example we assume a [TextRecognizer](text-recognition-api-ref.md#microsoftwindowsvisiontextrecognitiontextrecognizer-class) object has already been created and passed in to the function.
 
+```csharp
```

Comment:
@Karl-Bridge-Microsoft  is there a way to collapse the C# and C++ samples together with a toggle for the developer to see their chosen one at a time? I've seen this in other docs.

Reply:
@jwmsft since Karl is OOF